content/guides/hosting/iam/access-management/manage-organization.md

```diff
@@ -112,9 +112,47 @@ Use the W&B Console to turn off auto provisioning with SSO:
 {{< /tabpane >}}
 
 {{% alert title="" %}}
-Auto provisioning with SSO is useful for adding users to an organization at scale because organization administrators do not need to generate individual user invitations.
+Auto provisioning with SSO is useful for adding users to an organization at scale because organization admins do not need to generate individual user invitations.
 {{% /alert %}}
 
+### Create custom roles
+{{% alert %}}
+An Enterprise license is required to create or assign custom roles on Dedicated cloud or Self-managed deployments.
+{{% /alert %}}
+
+Custom roles allow organization admins to compose new roles by inheriting from the preceding View-Only or Member roles, and adding additional permissions to achieve fine-grained access control. Team admins can then assign any of those custom roles to users in their respective teams. Custom roles are created at the organization level but are assigned at the team level.
```

Comment:
This sentence:

"Custom roles allow organization admins to compose new roles by inheriting from the preceding View-Only or Member roles, and adding additional permissions to achieve fine-grained access control." 

Confused me. Maybe split into two sentences and clarify that bit?

Specifically this bit here: "...inheriting from the preceding View-Only or Member roles..." Not sure what that means...